src/main/js/core/Shape/shapeDefinitions.js

```diff
@@ -86,8 +86,8 @@ export const TEAR_ALT = {
  * @type {{path: {d: string}, options: {x: number, y: number, scale: number}}}
  */
 export const DIAMOND = {
-    path: { d: "M24 0l14 24-14 24-14-24L24 0z" },
-    options: { x: -8, y: -8, scale: 0.25 }
+    path: { d: "M24 0l12 24-12 24-12-24L24 0z" },
+    options: { x: -8, y: -8, scale: 0.34 }
```

Comment:
Currently, there are no tests for these changes.

So, What would be the testing consideration for these changes from the development perspective before we release this bug?

Reply:
I think there is a plan to add wdio testing to carbon-graphs. But I don't know when we are going to implement it.

cc / @ShettyAkarsh

Reply:
Yes, @pranav300  can you add the screenshot of before and after changes ?